client/src/site-search/search-results.tsx

```diff
@@ -286,93 +288,93 @@ function Results({
   const [searchParams] = useSearchParams();
 
   return (
-    <div>
-      <div className="search-results">
-        <p>
-          Found <ShowTotal total={metadata.total} /> in {metadata.took_ms}{" "}
-          milliseconds.
-        </p>
-
-        {!!suggestions.length && (
-          <div className="suggestions">
-            <p>Did you mean...</p>
-            <ul>
-              {suggestions.map((suggestion) => {
-                return (
-                  <li key={suggestion.text}>
-                    <Link
-                      to={`?${appendURL(searchParams, {
-                        q: suggestion.text,
-                        page: undefined,
-                      })}`}
-                    >
-                      {suggestion.text}
-                    </Link>{" "}
-                    <ShowTotal total={suggestion.total} />
-                  </li>
-                );
-              })}
-            </ul>
-          </div>
-        )}
+    <div className="search-results">
+      <p>
+        Found <ShowTotal total={metadata.total} /> in {metadata.took_ms}{" "}
+        milliseconds.
+      </p>
 
+      {!!suggestions.length && (
+        <div className="search-suggestions">
+          <p>Did you mean...</p>
+          <ul className="search-suggestion-list">
+            {suggestions.map((suggestion) => {
+              return (
+                <li key={suggestion.text}>
+                  <Link
+                    to={`?${appendURL(searchParams, {
+                      q: suggestion.text,
+                      page: undefined,
+                    })}`}
+                  >
+                    {suggestion.text}
+                  </Link>{" "}
+                  <ShowTotal total={suggestion.total} />
+                </li>
+              );
+            })}
+          </ul>
+        </div>
+      )}
+
+      <ul className="search-results-list">
         {documents.map((document) => {
           const highlights = document.highlight.body || [];
           return (
-            <div key={document.mdn_url}>
-              <p>
-                {/* We're using plain <a href> instead of <Link to> here until
+            <li
+              className="search-result-entry readable-line-length"
+              key={document.mdn_url}
+            >
+              {/* We're using plain <a href> instead of <Link to> here until
                 the bug has been figured out about scrolling to the top on click. */}
-                {document.highlight.title && document.highlight.title.length ? (
-                  <a
-                    className="title"
-                    href={document.mdn_url}
+              {document.highlight.title && document.highlight.title.length ? (
+                <a className="title" href={document.mdn_url}>
+                  <h3
```

Comment:
I find this a bit strange. But perhaps I just don't understand the reasoning. `a` is an inline element and `h3` is a block element. You don't write:
```
<a href="/"><p>bla bla</p></a>
```
so why do we wrap the `h3` here?